### ISSUE TITLE
Ajouter le téléchargement du fichier 3D d'une religion

### ISSUE THREAD

**Issue by ErenHistarion** [eveoniris/larpManager-php#441]
Une religion http://localhost:8080/religion/2/update doit avoir un nouveau champ pour y enregistrer un fichier 3D du symbole de son dieu.

Un joueur doit être capable de télécharger ce fichier depuis sa page personnage s'il est pretre.

**Comment by ErenHistarion** [comment]
@Gectou4 J'ai reçu les fichiers, ce sont de gros psd bien lourds.

**Comment by Gectou4** [comment]
@ErenHistarion 
c'est pas trop des fichier 3D du coups :D où alors de 3D dans toshop (hérésie!) 

Après tu as accès sftp donc il suffit de connaitre la capacité du serveur pour savoir si cela peut y être hébergé. Tu peux le mettre dans un des dossier de /private et on doit pouvoir map le path vers le fichier comme c'est fait pour les asset de class de personnage :)

Tips: les psd on doit pouvoir les zip, ça devrait aider.

**Comment by ErenHistarion** [comment]
Oui, j'avais pensé à ça... mais surtout, on va d'abord convertir les psd (oui, c'est de la 3D made in Photoshop) en stl pour que ce soit exploitable par le plus grand monde.

**Comment by ErenHistarion** [comment]
Les stl sont disponibles !

Il faudrait que les joueurs qui ont la compétence Prêtrise seulement puisse le télécharger sur leur fiche personnage.
